ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/cli/GetTapesView.java

```diff
@@ -58,6 +58,10 @@ public String render(final GetTapesResult obj) {
             bucketArray[4] = nullGuardToString(tape.getAvailableRawCapacity());
             bucketArray[5] = nullGuardToString(tape.getBucketId());
             bucketArray[6] = nullGuardToString(tape.getAssignedToStorageDomain());
+            bucketArray[6] = nullGuardToString(tape.getAssignedToStorageDomain());
+            bucketArray[7] = nullGuardToString(tape.getEjectDate());
```

Comment:
If you use nullGuardToDate, it will use the default DATE_FORMAT object and you get UTC, this gets you local.